pilot/pkg/serviceregistry/kube/controller/multicluster.go

```diff
@@ -219,40 +218,39 @@ func (m *Multicluster) ClusterAdded(cluster *multicluster.Cluster, clusterStopCh
 					// recreate it again.
 					client.RunAndWait(clusterStopCh)
 					nc.Run(leaderStop)
-				}).Run(clusterStopCh)
+				})
+			// Set up injection webhook patching for remote clusters we are controlling.
+			// The local cluster has this patching set up elsewhere. We may eventually want to move it here.
+			if features.ExternalIstiod && !localCluster && m.caBundleWatcher != nil {
```

Comment:
Remove guard from here if we delete patching setup in bootstrap/sidecarinjector.go.
```suggestion
			if m.caBundleWatcher != nil {
```